File: Core Java/tictactoe-app/src/com/techlab/tictactoe/Cell.java
package com.techlab.tictactoe;

import com.techlabs.enums.Seed;

public class Cell {

	Seed content;
	int row, col;

	public Cell(int row, int col) {
		this.row = row;
		this.col = col;
		clear();
	}

	public void clear() {
		content = Seed.EMPTY;
	}

	public char printCell() {
		switch (content) {
		case CROSS:
			return 'X';
		case NOUGHT:
			return 'O';
		case EMPTY:
			return ' ';
		}
		return ' ';
	}

	/*
	 * public void printCell() { switch (content) { case CROSS:
	 * System.out.print(" X "); break; case NOUGHT: System.out.print(" O ");
	 * break; case EMPTY: System.out.print("  "); break; } }
	 */
	public Seed getContent() {
		return content;
	}

	public void setContent(Seed content) {
		this.content = content;
	}
}

File: DOTNET/C#/TicTacToeApp/TicTacToeApp/Cell.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TicTacToeApp
{
    public enum GameStatus
    {
        PLAYING, DRAW, CROSS_WON, NOUGHT_WON
    }

    public enum Move
    {
        EMPTY, CROSS, NOUGHT
    }

    public class Cell
    {
        Move content;
        private int row, col;

        public Cell(int row, int col)
        {
            this.row = row;
            this.col = col;
            Clear();
        }

        public void Clear()
        {
            content = Move.EMPTY;
        }

        public char PrintCell()
        {
            switch (content)
            {
                case CROSS:
                    return 'X';
                case NOUGHT:
                    return 'O';
                case EMPTY:
                    return ' ';
            }
            return ' ';
        }

        public Move Content
        {
            get
            {
                return content;
            }
            set
            {
                content = value;
            }
        }
    }
}
